modules/ROOT/pages/dataweave-type-reuse-json-schema.adoc

```diff
@@ -0,0 +1,260 @@
+= Reusing Types from a JSON Schema
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+JSON schema is a standard providing a format for what JSON data is required for a given application and how to interact with it. Many existing data formats or types are expressed using this standard. In particular, REST APIs described by a RAML file are likely to have their types defined in JSON schemas.
```

Comment:
```suggestion
JSON Schema is a standard that provides a format for the JSON data required for a given application and how to interact with it. Many existing data formats or types are expressed using this standard. In particular, REST APIs described by RAML files are likely to have their types defined in JSON schemas.
```

Reply:
Wondering if there's a way to tighten the passive voice here.

Reply:
Not sure if Schema should be init cap. A quick google suggests yes.

Reply:
Reworded. First JSON Schema capitalized.